iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -2060,11 +2060,11 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
```

Comment:
somehow unrelated strings are overwritten;
its likely as issue with your local twine version;
please ensure your twine submodule is up-to-date (should point to `a9a97d1 Use custom om suffix for twine version`), remove the locally installed version (check via `gem list`) and remove the `tools/twine/twine-*.gem` file and then re-run `generate_localization.sh` - it'll install the right version

@RedAuburn had the same issue before